js/deer-config.js

```diff
@@ -6,6 +6,7 @@ export default {
     LABEL: "title", // attribute, alternate label for properties
     CONTEXT: "deer-context", // attribute, JSON-LD @context, scoped
     ATTRIBUTION: "deer-creator", // attribute, Web Annotation `creator`, scoped
+    MOTIVATION: "deer-motivation", // attribute, Web Annotation `motivation`, scoped
```

Comment:
I didn't see a change for evidence in the GitHub interface.  Upon viewing the file, I found it, so it must have already been there.  However, `EVIDENCE` is `nv-evidence`.  Is this on purpose?

Reply:
We can have more of a conversation around this, but `evidence` is not a DEER concept, more of an Eventities one. We can add it to DEER without too much trouble (and perhaps less than creating a context for Eventities) but I didn't want to shoehorn it into DEER or RERUM unilaterally.

It has always been there and we've never talked about it.